Tests/Test_Amperes_Task.c

```diff
@@ -133,10 +139,46 @@ void Task1(void *p_arg){
 	OSTaskDel(NULL, &err); // Delete task
 }
 
+//Task to prevent watchdog from tripping
+void Task2(void *p_arg){
+    OS_ERR err;
+
+    OSSemPost(&SafetyCheck_Sem4,      //Set semaphore once since Amperes Task doesn't run
```

Comment:
Wouldn't these semPosts be for Voltage, Open Wire, and temperature since we are testing current? Should there be 3 posts?

Reply:
Yeah, this is a copy-paste from the voltage and temperature test plan I can fix this